core/api/pod-api/src/spec.ts

```diff
@@ -140,7 +140,7 @@ export class PodSpec {
 
                             await polyOut.writeFile(path, content, { encoding: "utf-8" });
                             const filesWithPath = (await polyOut.readdir(this.path)).map(
-                                (path) => this.path + "/" + path
+                                (path) => this.path + "/" + path["id"]
```

Comment:
Really, this is exactly why I don't want core and feature code modified at the same time. Every one of this changes needs to be tested, and I would really feel more comfortable if you would create a specific PR for anything in the core modules.

Reply:
I see, I tried doing them on the same PR since adding the new feature and seeing whether it works would require changes in both. I will try to figure out why this is not working as expected :)

Reply:
_spec_ is just tests to begin with - the build/tests should be green before we land this, but at the end of the day, _spec_ is just legacy code that's not being used anywhere. Or more specifically, it's tests for code that's not being used anywhere.

Reply:
I agree, we could write some tests against `BrowserPod` in podjs, I suppose? That'd allow us to get rid of the default Pod implementation (which is not being used) and the tests for it. Sounds like something for another PR though!